Sources/Public/Animation/LottieAnimationView.swift

```diff
@@ -1390,6 +1390,13 @@ final public class LottieAnimationView: LottieAnimationViewBase {
         }
       }
 
+      // If attempting to play a zero-duration animation, just pause on that single frame instead
+      if animationContext.playFrom == animationContext.playTo {
```

Comment:
This change was also required to fix https://github.com/airbnb/lottie-ios/issues/1877 -- the logic here was causing us to attempt to play a zero-duration animation, which seems to not be supported properly by the Core Animation engine. Instead we can just pause the animation at that frame.